### PR TITLE
chore: fix test by normalizing stdout

### PR DESCRIPTION
**Pull request by lmiller1990**
- Closes N/A

### User facing changelog

N/A

### Additional details

Test snapshot hard codes Cypress version, so it will always fail. Now it normalizes it correctly, so it won't fail as we release new Cypress versions.

### Steps to test

Observe CI is passing.

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/server/test/integration/cypress_spec.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const Promise = require('bluebird')
 const electron = require('electron')
 const commitInfo = require('@cypress/commit-info')
 const Fixtures = require('@tooling/system-tests')
+const SystemTests = require('@tooling/system-tests/lib/system-tests')
```
Comment (flotwig) [inline packages/server/test/integration/cypress_spec.js:10]: I think this `require` changes the process's working directory, which is why these tests are failing: https://app.circleci.com/pipelines/github/cypress-io/cypress/40774/workflows/983f4e5c-d12e-488f-9e43-c455632b2da0/jobs/1683656

The `process.cwd` call could be relocated, or `normalizeStdout` could be moved to a module that doesn't call `process.cwd`.

**Review by flotwig — CHANGES_REQUESTED** [review]

File: packages/server/test/integration/cypress_spec.js (hunk 1 above)
Comment (lmiller1990) [reply packages/server/test/integration/cypress_spec.js:10]: Yep, was just looking into why this was failing. I'll make that refactor.

Ideally I'd have liked to have written this in `system-tests`, but it's quite hard to get the stdout to the correct state w/o a lot of stubbing, which system tests don't support for obvious reasons.

Comment (lmiller1990) [reply packages/server/test/integration/cypress_spec.js:10]: Lol, hate to see it: https://github.com/cypress-io/cypress/blob/develop/system-tests/lib/system-tests.ts#L31. Always good to fix a TODO.

Comment (lmiller1990) [reply packages/server/test/integration/cypress_spec.js:10]: Actually, problem is here: https://github.com/cypress-io/cypress/blob/develop/system-tests/lib/spec_helper.js (somewhere). I will find it.

Comment (lmiller1990) [reply packages/server/test/integration/cypress_spec.js:10]: This is to blame for at least some of the problems: https://github.com/cypress-io/cypress/blob/develop/system-tests/lib/spec_helper.js#L83-L86

Comment (lmiller1990) [reply packages/server/test/integration/cypress_spec.js:10]: I think we probably **don't** want all these side effects in `packages/server`. I'll just move the normalizeStdout function to a file, without any side effects.

File: system-tests/lib/normalizeStdout.ts [hunk 2]
```diff
@@ -0,0 +1,151 @@
+import Fixtures from './fixtures'
+import _ from 'lodash'
```
Comment (lmiller1990) [inline system-tests/lib/normalizeStdout.ts:2]: Copy pasted and threw in some types

File: packages/server/test/integration/cypress_spec.js (hunk 1 above)
Comment (lmiller1990) [reply packages/server/test/integration/cypress_spec.js]: Ok, I did it.

**Pull request merged**